react/ConditionContext.ts

```diff
@@ -4,7 +4,9 @@ import Noop from './Noop'
 
 type Action<K, V = void> = V extends void ? { type: K } : { type: K } & V
 
-type Actions = Action<'UPDATE_MATCH', { payload: { matches: boolean } }>
+type Actions =
+  | Action<'UPDATE_MATCH', { payload: { matches: boolean | null } }>
```

Comment:
I think this should be `undefined` instead of `null` right?
```suggestion
  | Action<'UPDATE_MATCH', { payload: { matches: boolean | undefined } }>
```